etc/PackageConfigurations/eos_package_description.json

```diff
@@ -1,520 +1,118 @@
-
-        {"src":"Assets/Scripts/OpenID/*", "dest": "Samples~/Samples/StandardSamples/Scripts/OpenID/"},
-        {"src":"Assets/Scripts/OpenID/CloudFunction/*", "dest": "Samples~/Samples/StandardSamples/Scripts/OpenID/CloudFunction/"},
-
-       {"src":"Assets/Scripts/SignInWithApple/*", "dest": "Samples~/Samples/StandardSamples/Scripts/SignInWithApple/"},
-        {"src":"Assets/Scripts/SignInWithApple/Editor/*", "dest": "Samples~/Samples/StandardSamples/Scripts/SignInWithApple/Editor/"},
-
-        {"src":"Assets/Scenes/StandardSamples/*.unity", "dest": "Samples~/Samples/StandardSamples/Scenes/"},
-        {"src":"Assets/Scenes/SampleScenes.asset", "dest": "Samples~/Samples/StandardSamples/Scenes/"},
-        {"src":"Assets/Materials/StandardSamplesMaterials/*", "dest": "Samples~/Samples/StandardSamples/Materials/"},
-
-        {"comment": "-------------------------------P2P NetCode Sample-----------------------------------------"},
-
-        {"src":"Assets/Scenes/P2PNetcodeSample/*.unity", "dest": "Samples~/Samples/P2PNetcodeSample/Scenes/"},
-        {"src":"Assets/Materials/NetcodeMaterials/*", "dest": "Samples~/Samples/P2PNetcodeSample/Materials/"},
-        {"src":"Assets/Scripts/Networking/*.cs", "dest": "Samples~/Samples/P2PNetcodeSample/Scripts/Networking/"},
-        {"src":"Assets/Scripts/UI/P2PNetcode/*", "dest": "Samples~/Samples/P2PNetcodeSample/Scripts/UI/"},
-
-        {"comment": "-------------------------------End P2P NetCode Sample-----------------------------------------"},
-
-	{"comment": "-------------------------------Performance Stress Test Sample-----------------------------------------"},
-
-        {"src":"Assets/Scenes/PerformanceStressTestSample/*.unity", "dest": "Samples~/Samples/PerformanceStressTestSample/Scenes/"},
-        {"src":"Assets/Materials/PerformanceStressTestMaterials/*", "dest": "Samples~/Samples/PerformanceStressTestSample/Materials/"},
-        {"src":"Assets/Scripts/PostProcessing/*", "dest": "Samples~/Samples/PerformanceStressTestSample/Scripts/PostProcessing/"},
-        {"src":"Assets/Resources/PerformanceStressTest_Profiles/*", "dest": "Samples~/Samples/PerformanceStressTestSample/Profiles/"},
-
-	{"comment": "-------------------------------End Performance Stress Test Sample-----------------------------------------"},
-
-        {"src":"Assets/Readme/*.txt", "dest": "Samples~/Samples/StandardSamples/Readme/"}
+        {"src":"Assets/Plugins/Standalone/Editor/*",                             "dest": "Editor/Standalone/"},
+        {"src":"Assets/Plugins/Standalone/Editor/anticheat_integritytool.cfg",   "dest": "Editor/Standalone/"},
+
+        {"src":"tools/bin/*",                                                    "dest": "bin~/", "recurse": true},
+
+        {"src":"Assets/link.xml",                                                "dest": "Editor/"},
+
+        {"src":"Assets/Plugins/Source/EOS_SDK/*",                                "dest": "Runtime/EOS_SDK/", "recurse": true },
```

Comment:
Remove the `recurse` from this for the time being - instead explicitly ask to include each of the directories inside the `Generated` directory.